tests/bats/_helpers_govcms.bash

```diff
@@ -84,13 +84,16 @@ prepare_fixture_dir(){
 }
 
 download_code_from_github(){
+  local release_url
   local user="${1?Required value}"
   local repo="${2?Required value}"
-  curl --silent "https://api.github.com/repos/$user/$repo/releases/latest" |
-    grep '"tag_name":' |
-    sed -E 's/.*"([^"]+)".*/\1/' |
-    xargs -I {} curl -sL "https://github.com/$user/$repo/archive/"{}'.tar.gz' |
-    tar xvz -C "$(pwd)" --strip 1
+
+  release_url=$(curl -sL -H "Authorization: token $GOVCMS_GITHUB_TOKEN" "https://api.github.com/repos/$user/$repo/releases/latest" | jq -r '.tarball_url')
+
+  debug "$release_url"
```

Comment:
Just confirming that this debug line was left here intentionally.

Reply:
Intentionally left - but we can probably remove the entire helper function as it not used anymore.